api/src/backend/api/migrations/0006_findings_first_seen.py

```diff
@@ -0,0 +1,58 @@
+from django.db import migrations, models
+
+
+def populate_first_seen(apps, schema_editor):
```

Comment:
Do we want to backfill all previous findings? This blocks the app startup while updating all the findings.